

<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../fastani/">
      
      
        <link rel="next" href="../prokka-gbk/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.18">
    
    
      
        <title>checkm - BGCFlow Report</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.66ac8b77.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#checkm" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="BGCFlow Report" class="md-header__button md-logo" aria-label="BGCFlow Report" data-md-component="logo">
      
  <img src="../assets/bgcflow/BGCFlow_logo.svg" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            BGCFlow Report
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              checkm
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="blue" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="blue" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../seqfu/" class="md-tabs__link">
          
  
  QC and Data Selection

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../prokka-gbk/" class="md-tabs__link">
          
  
  Functional Annotation

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../antismash/" class="md-tabs__link">
          
  
  Genome Mining

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../automlst-wrapper/" class="md-tabs__link">
          
  
  Phylogenomic Placement

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../roary/" class="md-tabs__link">
          
  
  Comparative Genomics

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


  

<nav class="md-nav md-nav--primary md-nav--lifted md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="BGCFlow Report" class="md-nav__button md-logo" aria-label="BGCFlow Report" data-md-component="logo">
      
  <img src="../assets/bgcflow/BGCFlow_logo.svg" alt="logo">

    </a>
    BGCFlow Report
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
      
        
        
      
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    QC and Data Selection
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            QC and Data Selection
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../seqfu/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    seqfu
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../mash/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    mash
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../fastani/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    fastani
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    checkm
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    checkm
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#description" class="md-nav__link">
    <span class="md-ellipsis">
      Description
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#result" class="md-nav__link">
    <span class="md-ellipsis">
      Result
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#references" class="md-nav__link">
    <span class="md-ellipsis">
      References
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Functional Annotation
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Functional Annotation
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../prokka-gbk/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    prokka-gbk
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../deeptfactor/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    deeptfactor
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Genome Mining
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Genome Mining
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../antismash/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    antismash
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../query-bigslice/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    query-bigslice
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../bigscape/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    bigscape
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../bigslice/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    bigslice
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../arts/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    arts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../cblaster-genome/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    cblaster-genome
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../cblaster-bgc/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    cblaster-bgc
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../gecco/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    gecco
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Phylogenomic Placement
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Phylogenomic Placement
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../automlst-wrapper/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    automlst-wrapper
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
      
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Comparative Genomics
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Comparative Genomics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../roary/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    roary
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../eggnog-roary/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    eggnog-roary
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                



                  


<h1 id="checkm">CheckM</h1>
<p>Summary of <a href="https://github.com/Ecogenomics/CheckM">CheckM</a> results from project: <code>[mq_saccharopolyspora]</code> </p>
<h2 id="description">Description</h2>
<p>Assess the quality of microbial genomes recovered from isolates, single cells, and metagenomes</p>
<h2 id="result">Result</h2>
<pre><code>high_quality=25, medium_quality=1, low_quality=0, contaminated=0, unknown=0
</code></pre>
<style>
  #altair-viz-e4d21f8f5279454ca734e421fb1a022d.vega-embed {
    width: 100%;
    display: flex;
  }

  #altair-viz-e4d21f8f5279454ca734e421fb1a022d.vega-embed details,
  #altair-viz-e4d21f8f5279454ca734e421fb1a022d.vega-embed details summary {
    position: relative;
  }
</style>
<div id="altair-viz-e4d21f8f5279454ca734e421fb1a022d"></div>
<script type="text/javascript">
  var VEGA_DEBUG = (typeof VEGA_DEBUG == "undefined") ? {} : VEGA_DEBUG;
  (function(spec, embedOpt){
    let outputDiv = document.currentScript.previousElementSibling;
    if (outputDiv.id !== "altair-viz-e4d21f8f5279454ca734e421fb1a022d") {
      outputDiv = document.getElementById("altair-viz-e4d21f8f5279454ca734e421fb1a022d");
    }
    const paths = {
      "vega": "https://cdn.jsdelivr.net/npm/vega@5?noext",
      "vega-lib": "https://cdn.jsdelivr.net/npm/vega-lib?noext",
      "vega-lite": "https://cdn.jsdelivr.net/npm/vega-lite@5.16.3?noext",
      "vega-embed": "https://cdn.jsdelivr.net/npm/vega-embed@6?noext",
    };

    function maybeLoadScript(lib, version) {
      var key = `${lib.replace("-", "")}_version`;
      return (VEGA_DEBUG[key] == version) ?
        Promise.resolve(paths[lib]) :
        new Promise(function(resolve, reject) {
          var s = document.createElement('script');
          document.getElementsByTagName("head")[0].appendChild(s);
          s.async = true;
          s.onload = () => {
            VEGA_DEBUG[key] = version;
            return resolve(paths[lib]);
          };
          s.onerror = () => reject(`Error loading script: ${paths[lib]}`);
          s.src = paths[lib];
        });
    }

    function showError(err) {
      outputDiv.innerHTML = `<div class="error" style="color:red;">${err}</div>`;
      throw err;
    }

    function displayChart(vegaEmbed) {
      vegaEmbed(outputDiv, spec, embedOpt)
        .catch(err => showError(`Javascript Error: ${err.message}<br>This usually means there's a typo in your chart specification. See the javascript console for the full traceback.`));
    }

    if(typeof define === "function" && define.amd) {
      requirejs.config({paths});
      require(["vega-embed"], displayChart, err => showError(`Error loading script: ${err.message}`));
    } else {
      maybeLoadScript("vega", "5")
        .then(() => maybeLoadScript("vega-lite", "5.16.3"))
        .then(() => maybeLoadScript("vega-embed", "6"))
        .catch(showError)
        .then(() => displayChart(vegaEmbed));
    }
  })({"config": {"view": {"continuousWidth": 500, "continuousHeight": 500}, "axis": {"labelFontSize": 10, "titleFontSize": 12}, "legend": {"labelFontSize": 10, "titleFontSize": 12}}, "data": {"name": "data-b06ecd19933f70535d57348225771973"}, "mark": {"type": "point", "filled": true, "opacity": 0.8, "size": 100, "stroke": "black", "strokeWidth": 0.5}, "encoding": {"color": {"field": "genome_quality", "type": "nominal"}, "tooltip": [{"field": "genome_id", "type": "nominal"}, {"field": "Completeness", "type": "quantitative"}, {"field": "Contamination", "type": "quantitative"}, {"field": "GC", "type": "quantitative"}, {"field": "GC std", "type": "quantitative"}, {"field": "Genome size", "type": "quantitative"}, {"field": "# ambiguous bases", "type": "quantitative"}, {"field": "# scaffolds", "type": "quantitative"}, {"field": "# contigs", "type": "quantitative"}, {"field": "Longest scaffold", "type": "quantitative"}, {"field": "Longest contig", "type": "quantitative"}, {"field": "N50 (scaffolds)", "type": "quantitative"}, {"field": "N50 (contigs)", "type": "quantitative"}, {"field": "Mean scaffold length", "type": "quantitative"}, {"field": "Mean contig length", "type": "quantitative"}, {"field": "Coding density", "type": "quantitative"}, {"field": "Translation table", "type": "quantitative"}, {"field": "# predicted genes", "type": "quantitative"}, {"field": "genome_quality", "type": "nominal"}], "x": {"field": "Completeness", "scale": {"domain": [97.34042553191487, 100]}, "title": "Completeness (%)", "type": "quantitative"}, "y": {"field": "Contamination", "scale": {"domain": [0, 6.94]}, "title": "Contamination (%)", "type": "quantitative"}}, "$schema": "https://vega.github.io/schema/vega-lite/v5.16.3.json", "datasets": {"data-b06ecd19933f70535d57348225771973": [{"genome_id": "GCF_003931915.1", "Completeness": 100.0, "Contamination": 1.4893617021276595, "GC": 0.7063295616186468, "GC std": 0.0439092988084981, "Genome size": 5824618, "# ambiguous bases": 123, "# scaffolds": 35, "# contigs": 36, "Longest scaffold": 768785, "Longest contig": 768785, "N50 (scaffolds)": 345543, "N50 (contigs)": 345543, "Mean scaffold length": 166417.65714285715, "Mean contig length": 161791.52777777778, "Coding density": 0.8596021232637059, "Translation table": 11, "# predicted genes": 5362, "genome_quality": "high_quality"}, {"genome_id": "GCF_013410345.1", "Completeness": 99.645390070922, "Contamination": 1.8439716312056735, "GC": 0.7247914414760197, "GC std": 0.0, "Genome size": 5783988, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 5783988, "Longest contig": 5783988, "N50 (scaffolds)": 5783988, "N50 (contigs)": 5783988, "Mean scaffold length": 5783988.0, "Mean contig length": 5783988.0, "Coding density": 0.87912855282549, "Translation table": 11, "# predicted genes": 5267, "genome_quality": "high_quality"}, {"genome_id": "GCF_016526145.1", "Completeness": 99.46808510638296, "Contamination": 1.4775413711583922, "GC": 0.7029273240831624, "GC std": 0.0332832008963936, "Genome size": 6397171, "# ambiguous bases": 100, "# scaffolds": 34, "# contigs": 35, "Longest scaffold": 979558, "Longest contig": 979558, "N50 (scaffolds)": 454807, "N50 (contigs)": 406614, "Mean scaffold length": 188152.08823529413, "Mean contig length": 182773.45714285717, "Coding density": 0.8722302405235064, "Translation table": 11, "# predicted genes": 5826, "genome_quality": "high_quality"}, {"genome_id": "GCF_007829955.1", "Completeness": 97.34042553191487, "Contamination": 3.882978723404255, "GC": 0.6970017271329593, "GC std": 0.0044569940280547, "Genome size": 6410624, "# ambiguous bases": 0, "# scaffolds": 8, "# contigs": 8, "Longest scaffold": 1972629, "Longest contig": 1972629, "N50 (scaffolds)": 1390797, "N50 (contigs)": 1390797, "Mean scaffold length": 801328.0, "Mean contig length": 801328.0, "Coding density": 0.8865547254058264, "Translation table": 11, "# predicted genes": 5786, "genome_quality": "high_quality"}, {"genome_id": "GCF_014203325.1", "Completeness": 99.46808510638296, "Contamination": 1.536643026004728, "GC": 0.7110368743562213, "GC std": 0.0, "Genome size": 6825909, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 6825909, "Longest contig": 6825909, "N50 (scaffolds)": 6825909, "N50 (contigs)": 6825909, "Mean scaffold length": 6825909.0, "Mean contig length": 6825909.0, "Coding density": 0.8612406054636825, "Translation table": 11, "# predicted genes": 5929, "genome_quality": "high_quality"}, {"genome_id": "GCF_012277335.1", "Completeness": 99.88179669030734, "Contamination": 3.050958760178619, "GC": 0.6810451071301798, "GC std": 0.0, "Genome size": 8045912, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 8045912, "Longest contig": 8045912, "N50 (scaffolds)": 8045912, "N50 (contigs)": 8045912, "Mean scaffold length": 8045912.0, "Mean contig length": 8045912.0, "Coding density": 0.8422994186364454, "Translation table": 11, "# predicted genes": 7504, "genome_quality": "high_quality"}, {"genome_id": "GCF_008630535.1", "Completeness": 99.43853427895982, "Contamination": 2.624113475177305, "GC": 0.7140299510212214, "GC std": 0.0483380434393586, "Genome size": 7549392, "# ambiguous bases": 0, "# scaffolds": 46, "# contigs": 46, "Longest scaffold": 688257, "Longest contig": 688257, "N50 (scaffolds)": 504440, "N50 (contigs)": 504440, "Mean scaffold length": 164117.21739130435, "Mean contig length": 164117.21739130435, "Coding density": 0.8811830144732185, "Translation table": 11, "# predicted genes": 6830, "genome_quality": "high_quality"}, {"genome_id": "GCF_016859185.1", "Completeness": 100.0, "Contamination": 2.6549776727081693, "GC": 0.7111245450688278, "GC std": 0.0, "Genome size": 8305652, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 8305652, "Longest contig": 8305652, "N50 (scaffolds)": 8305652, "N50 (contigs)": 8305652, "Mean scaffold length": 8305652.0, "Mean contig length": 8305652.0, "Coding density": 0.8626696615750334, "Translation table": 11, "# predicted genes": 7407, "genome_quality": "high_quality"}, {"genome_id": "GCF_000062885.1", "Completeness": 100.0, "Contamination": 1.9457578145521408, "GC": 0.7114596048487697, "GC std": 0.0, "Genome size": 8212805, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 8212805, "Longest contig": 8212805, "N50 (scaffolds)": 8212805, "N50 (contigs)": 8212805, "Mean scaffold length": 8212805.0, "Mean contig length": 8212805.0, "Coding density": 0.8636916376317226, "Translation table": 11, "# predicted genes": 7279, "genome_quality": "high_quality"}, {"genome_id": "GCF_014203395.1", "Completeness": 99.29078014184395, "Contamination": 4.078014184397163, "GC": 0.6772580746183299, "GC std": 0.0103248041079479, "Genome size": 8541735, "# ambiguous bases": 0, "# scaffolds": 6, "# contigs": 6, "Longest scaffold": 5889697, "Longest contig": 5889697, "N50 (scaffolds)": 5889697, "N50 (contigs)": 5889697, "Mean scaffold length": 1423622.5, "Mean contig length": 1423622.5, "Coding density": 0.8655130368713148, "Translation table": 11, "# predicted genes": 7445, "genome_quality": "high_quality"}, {"genome_id": "GCF_000194155.1", "Completeness": 98.64066193853428, "Contamination": 2.9977672708169165, "GC": 0.6805661874795961, "GC std": 0.0314472022848502, "Genome size": 8581920, "# ambiguous bases": 54144, "# scaffolds": 22, "# contigs": 117, "Longest scaffold": 3851573, "Longest contig": 471503, "N50 (scaffolds)": 2392754, "N50 (contigs)": 163471, "Mean scaffold length": 390087.2727272727, "Mean contig length": 72886.97435897436, "Coding density": 0.8275158705744169, "Translation table": 11, "# predicted genes": 8230, "genome_quality": "high_quality"}, {"genome_id": "GCF_002564065.1", "Completeness": 100.0, "Contamination": 1.9457578145521408, "GC": 0.7114145473027494, "GC std": 0.0, "Genome size": 8230103, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 8230103, "Longest contig": 8230103, "N50 (scaffolds)": 8230103, "N50 (contigs)": 8230103, "Mean scaffold length": 8230103.0, "Mean contig length": 8230103.0, "Coding density": 0.8635506748821977, "Translation table": 11, "# predicted genes": 7308, "genome_quality": "high_quality"}, {"genome_id": "GCF_003635025.1", "Completeness": 100.0, "Contamination": 3.1855791962174944, "GC": 0.7038623374794615, "GC std": 0.0012732815388598, "Genome size": 8266186, "# ambiguous bases": 0, "# scaffolds": 2, "# contigs": 2, "Longest scaffold": 8125734, "Longest contig": 8125734, "N50 (scaffolds)": 8125734, "N50 (contigs)": 8125734, "Mean scaffold length": 4133093.0, "Mean contig length": 4133093.0, "Coding density": 0.8751350380937473, "Translation table": 11, "# predicted genes": 7658, "genome_quality": "high_quality"}, {"genome_id": "GCF_014490055.1", "Completeness": 99.52718676122932, "Contamination": 3.245994221171526, "GC": 0.6796706110054318, "GC std": 0.0, "Genome size": 8876435, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 8876435, "Longest contig": 8876435, "N50 (scaffolds)": 8876435, "N50 (contigs)": 8876435, "Mean scaffold length": 8876435.0, "Mean contig length": 8876435.0, "Coding density": 0.8446922666588558, "Translation table": 11, "# predicted genes": 8022, "genome_quality": "high_quality"}, {"genome_id": "GCF_002846475.1", "Completeness": 99.52718676122932, "Contamination": 3.245994221171526, "GC": 0.6794950162105907, "GC std": 0.0, "Genome size": 9020646, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 9020646, "Longest contig": 9020646, "N50 (scaffolds)": 9020646, "N50 (contigs)": 9020646, "Mean scaffold length": 9020646.0, "Mean contig length": 9020646.0, "Coding density": 0.8479478077290695, "Translation table": 11, "# predicted genes": 8028, "genome_quality": "high_quality"}, {"genome_id": "GCF_014697215.1", "Completeness": 99.88179669030734, "Contamination": 5.9430935494765285, "GC": 0.6792499724193344, "GC std": 0.0089040739990278, "Genome size": 9608180, "# ambiguous bases": 0, "# scaffolds": 3, "# contigs": 3, "Longest scaffold": 9436548, "Longest contig": 9436548, "N50 (scaffolds)": 9436548, "N50 (contigs)": 9436548, "Mean scaffold length": 3202726.6666666665, "Mean contig length": 3202726.6666666665, "Coding density": 0.8417722190883185, "Translation table": 11, "# predicted genes": 9217, "genome_quality": "medium_quality"}, {"genome_id": "GCF_024734405.1", "Completeness": 99.822695035461, "Contamination": 1.536643026004728, "GC": 0.7266721423436433, "GC std": 0.0, "Genome size": 6344421, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 6344421, "Longest contig": 6344421, "N50 (scaffolds)": 6344421, "N50 (contigs)": 6344421, "Mean scaffold length": 6344421.0, "Mean contig length": 6344421.0, "Coding density": 0.8752644252328148, "Translation table": 11, "# predicted genes": 5540, "genome_quality": "high_quality"}, {"genome_id": "GCF_018070075.1", "Completeness": 99.822695035461, "Contamination": 2.861833464670344, "GC": 0.6975256232823224, "GC std": 0.0319032695050071, "Genome size": 7167017, "# ambiguous bases": 0, "# scaffolds": 34, "# contigs": 34, "Longest scaffold": 858004, "Longest contig": 858004, "N50 (scaffolds)": 363995, "N50 (contigs)": 363995, "Mean scaffold length": 210794.61764705885, "Mean contig length": 210794.61764705885, "Coding density": 0.8839261857478502, "Translation table": 11, "# predicted genes": 6643, "genome_quality": "high_quality"}, {"genome_id": "GCF_022828475.1", "Completeness": 99.05437352245865, "Contamination": 2.423167848699763, "GC": 0.6947996676695216, "GC std": 0.0, "Genome size": 6878695, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 6878695, "Longest contig": 6878695, "N50 (scaffolds)": 6878695, "N50 (contigs)": 6878695, "Mean scaffold length": 6878695.0, "Mean contig length": 6878695.0, "Coding density": 0.8527571000022534, "Translation table": 11, "# predicted genes": 6055, "genome_quality": "high_quality"}, {"genome_id": "GCF_900116135.1", "Completeness": 99.46808510638296, "Contamination": 1.75531914893617, "GC": 0.7077562954321726, "GC std": 0.0373222530498549, "Genome size": 6291299, "# ambiguous bases": 190, "# scaffolds": 22, "# contigs": 24, "Longest scaffold": 1461917, "Longest contig": 1461917, "N50 (scaffolds)": 505605, "N50 (contigs)": 453928, "Mean scaffold length": 285968.13636363635, "Mean contig length": 262129.54166666663, "Coding density": 0.8874002014528319, "Translation table": 11, "# predicted genes": 5631, "genome_quality": "high_quality"}, {"genome_id": "GCF_025643595.1", "Completeness": 99.645390070922, "Contamination": 2.848699763593381, "GC": 0.7170576832060522, "GC std": 0.0, "Genome size": 6562638, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 6562638, "Longest contig": 6562638, "N50 (scaffolds)": 6562638, "N50 (contigs)": 6562638, "Mean scaffold length": 6562638.0, "Mean contig length": 6562638.0, "Coding density": 0.882625858686705, "Translation table": 11, "# predicted genes": 6032, "genome_quality": "high_quality"}, {"genome_id": "GCF_022392385.1", "Completeness": 99.645390070922, "Contamination": 1.9457578145521408, "GC": 0.7118386770561067, "GC std": 0.0015522632597943, "Genome size": 8181083, "# ambiguous bases": 10094, "# scaffolds": 2, "# contigs": 15, "Longest scaffold": 5801759, "Longest contig": 1955749, "N50 (scaffolds)": 5801759, "N50 (contigs)": 1073489, "Mean scaffold length": 4090541.5, "Mean contig length": 544732.6, "Coding density": 0.8639395297664135, "Translation table": 11, "# predicted genes": 7263, "genome_quality": "high_quality"}, {"genome_id": "GCF_018141105.1", "Completeness": 99.645390070922, "Contamination": 2.2826372471762544, "GC": 0.710667418576433, "GC std": 0.0, "Genome size": 8243897, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 8243897, "Longest contig": 8243897, "N50 (scaffolds)": 8243897, "N50 (contigs)": 8243897, "Mean scaffold length": 8243897.0, "Mean contig length": 8243897.0, "Coding density": 0.8728290758606033, "Translation table": 11, "# predicted genes": 7435, "genome_quality": "high_quality"}, {"genome_id": "GCF_900108315.1", "Completeness": 99.645390070922, "Contamination": 2.724586288416076, "GC": 0.7089549423325838, "GC std": 0.0724139671090498, "Genome size": 7744048, "# ambiguous bases": 4322, "# scaffolds": 27, "# contigs": 33, "Longest scaffold": 1906593, "Longest contig": 1077102, "N50 (scaffolds)": 729242, "N50 (contigs)": 469672, "Mean scaffold length": 286816.5925925926, "Mean contig length": 234537.15151515152, "Coding density": 0.8804439228682467, "Translation table": 11, "# predicted genes": 7185, "genome_quality": "high_quality"}, {"genome_id": "GCF_900112555.1", "Completeness": 99.52718676122932, "Contamination": 2.724586288416076, "GC": 0.7089995844725187, "GC std": 0.038333430857097, "Genome size": 7777251, "# ambiguous bases": 1592, "# scaffolds": 28, "# contigs": 31, "Longest scaffold": 1963067, "Longest contig": 1200418, "N50 (scaffolds)": 406024, "N50 (contigs)": 406024, "Mean scaffold length": 277758.96428571426, "Mean contig length": 250827.70967741936, "Coding density": 0.8810862604280099, "Translation table": 11, "# predicted genes": 7257, "genome_quality": "high_quality"}, {"genome_id": "GCF_900114905.1", "Completeness": 100.0, "Contamination": 3.1855791962174944, "GC": 0.7043312457011255, "GC std": 0.0367099615787713, "Genome size": 8219321, "# ambiguous bases": 600, "# scaffolds": 43, "# contigs": 47, "Longest scaffold": 1889235, "Longest contig": 1001818, "N50 (scaffolds)": 345124, "N50 (contigs)": 343399, "Mean scaffold length": 191147.0, "Mean contig length": 174866.40425531915, "Coding density": 0.8772927836739799, "Translation table": 11, "# predicted genes": 7662, "genome_quality": "high_quality"}]}}, {"mode": "vega-lite"});
</script>

<h2 id="references">References</h2>
<p><font size="2"></p>
<ul>
<li><em>Parks DH, Imelfort M, Skennerton CT, Hugenholtz P, Tyson GW. 2014. Assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. <a href="https://genome.cshlp.org/content/25/7/1043.long">Genome Research, 25: 1043-1055.</a></em></li>
</ul>
<p></font></p>












                

              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        <div class="md-social">
  
    
    
    
    
      
      
    
    <a href="https://twitter.com/NPGMgroup" target="_blank" rel="noopener" title="twitter.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg>
    </a>
  
    
    
    
    
      
      
    
    <a href="https://github.com/NBChub/bgcflow" target="_blank" rel="noopener" title="github.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "toc.integrate"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.3220b9d7.min.js"></script>
      
    
  </body>
</html>