# CheckM
Summary of [CheckM](https://github.com/Ecogenomics/CheckM) results from project: `[{{ project().name }}]` 

## Description
Assess the quality of microbial genomes recovered from isolates, single cells, and metagenomes

## Result

    high_quality=25, medium_quality=1, low_quality=0, contaminated=0, unknown=0






<style>
  #altair-viz-e4d21f8f5279454ca734e421fb1a022d.vega-embed {
    width: 100%;
    display: flex;
  }

  #altair-viz-e4d21f8f5279454ca734e421fb1a022d.vega-embed details,
  #altair-viz-e4d21f8f5279454ca734e421fb1a022d.vega-embed details summary {
    position: relative;
  }
</style>
<div id="altair-viz-e4d21f8f5279454ca734e421fb1a022d"></div>
<script type="text/javascript">
  var VEGA_DEBUG = (typeof VEGA_DEBUG == "undefined") ? {} : VEGA_DEBUG;
  (function(spec, embedOpt){
    let outputDiv = document.currentScript.previousElementSibling;
    if (outputDiv.id !== "altair-viz-e4d21f8f5279454ca734e421fb1a022d") {
      outputDiv = document.getElementById("altair-viz-e4d21f8f5279454ca734e421fb1a022d");
    }
    const paths = {
      "vega": "https://cdn.jsdelivr.net/npm/vega@5?noext",
      "vega-lib": "https://cdn.jsdelivr.net/npm/vega-lib?noext",
      "vega-lite": "https://cdn.jsdelivr.net/npm/vega-lite@5.16.3?noext",
      "vega-embed": "https://cdn.jsdelivr.net/npm/vega-embed@6?noext",
    };

    function maybeLoadScript(lib, version) {
      var key = `${lib.replace("-", "")}_version`;
      return (VEGA_DEBUG[key] == version) ?
        Promise.resolve(paths[lib]) :
        new Promise(function(resolve, reject) {
          var s = document.createElement('script');
          document.getElementsByTagName("head")[0].appendChild(s);
          s.async = true;
          s.onload = () => {
            VEGA_DEBUG[key] = version;
            return resolve(paths[lib]);
          };
          s.onerror = () => reject(`Error loading script: ${paths[lib]}`);
          s.src = paths[lib];
        });
    }

    function showError(err) {
      outputDiv.innerHTML = `<div class="error" style="color:red;">${err}</div>`;
      throw err;
    }

    function displayChart(vegaEmbed) {
      vegaEmbed(outputDiv, spec, embedOpt)
        .catch(err => showError(`Javascript Error: ${err.message}<br>This usually means there's a typo in your chart specification. See the javascript console for the full traceback.`));
    }

    if(typeof define === "function" && define.amd) {
      requirejs.config({paths});
      require(["vega-embed"], displayChart, err => showError(`Error loading script: ${err.message}`));
    } else {
      maybeLoadScript("vega", "5")
        .then(() => maybeLoadScript("vega-lite", "5.16.3"))
        .then(() => maybeLoadScript("vega-embed", "6"))
        .catch(showError)
        .then(() => displayChart(vegaEmbed));
    }
  })({"config": {"view": {"continuousWidth": 500, "continuousHeight": 500}, "axis": {"labelFontSize": 10, "titleFontSize": 12}, "legend": {"labelFontSize": 10, "titleFontSize": 12}}, "data": {"name": "data-b06ecd19933f70535d57348225771973"}, "mark": {"type": "point", "filled": true, "opacity": 0.8, "size": 100, "stroke": "black", "strokeWidth": 0.5}, "encoding": {"color": {"field": "genome_quality", "type": "nominal"}, "tooltip": [{"field": "genome_id", "type": "nominal"}, {"field": "Completeness", "type": "quantitative"}, {"field": "Contamination", "type": "quantitative"}, {"field": "GC", "type": "quantitative"}, {"field": "GC std", "type": "quantitative"}, {"field": "Genome size", "type": "quantitative"}, {"field": "# ambiguous bases", "type": "quantitative"}, {"field": "# scaffolds", "type": "quantitative"}, {"field": "# contigs", "type": "quantitative"}, {"field": "Longest scaffold", "type": "quantitative"}, {"field": "Longest contig", "type": "quantitative"}, {"field": "N50 (scaffolds)", "type": "quantitative"}, {"field": "N50 (contigs)", "type": "quantitative"}, {"field": "Mean scaffold length", "type": "quantitative"}, {"field": "Mean contig length", "type": "quantitative"}, {"field": "Coding density", "type": "quantitative"}, {"field": "Translation table", "type": "quantitative"}, {"field": "# predicted genes", "type": "quantitative"}, {"field": "genome_quality", "type": "nominal"}], "x": {"field": "Completeness", "scale": {"domain": [97.34042553191487, 100]}, "title": "Completeness (%)", "type": "quantitative"}, "y": {"field": "Contamination", "scale": {"domain": [0, 6.94]}, "title": "Contamination (%)", "type": "quantitative"}}, "$schema": "https://vega.github.io/schema/vega-lite/v5.16.3.json", "datasets": {"data-b06ecd19933f70535d57348225771973": [{"genome_id": "GCF_003931915.1", "Completeness": 100.0, "Contamination": 1.4893617021276595, "GC": 0.7063295616186468, "GC std": 0.0439092988084981, "Genome size": 5824618, "# ambiguous bases": 123, "# scaffolds": 35, "# contigs": 36, "Longest scaffold": 768785, "Longest contig": 768785, "N50 (scaffolds)": 345543, "N50 (contigs)": 345543, "Mean scaffold length": 166417.65714285715, "Mean contig length": 161791.52777777778, "Coding density": 0.8596021232637059, "Translation table": 11, "# predicted genes": 5362, "genome_quality": "high_quality"}, {"genome_id": "GCF_013410345.1", "Completeness": 99.645390070922, "Contamination": 1.8439716312056735, "GC": 0.7247914414760197, "GC std": 0.0, "Genome size": 5783988, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 5783988, "Longest contig": 5783988, "N50 (scaffolds)": 5783988, "N50 (contigs)": 5783988, "Mean scaffold length": 5783988.0, "Mean contig length": 5783988.0, "Coding density": 0.87912855282549, "Translation table": 11, "# predicted genes": 5267, "genome_quality": "high_quality"}, {"genome_id": "GCF_016526145.1", "Completeness": 99.46808510638296, "Contamination": 1.4775413711583922, "GC": 0.7029273240831624, "GC std": 0.0332832008963936, "Genome size": 6397171, "# ambiguous bases": 100, "# scaffolds": 34, "# contigs": 35, "Longest scaffold": 979558, "Longest contig": 979558, "N50 (scaffolds)": 454807, "N50 (contigs)": 406614, "Mean scaffold length": 188152.08823529413, "Mean contig length": 182773.45714285717, "Coding density": 0.8722302405235064, "Translation table": 11, "# predicted genes": 5826, "genome_quality": "high_quality"}, {"genome_id": "GCF_007829955.1", "Completeness": 97.34042553191487, "Contamination": 3.882978723404255, "GC": 0.6970017271329593, "GC std": 0.0044569940280547, "Genome size": 6410624, "# ambiguous bases": 0, "# scaffolds": 8, "# contigs": 8, "Longest scaffold": 1972629, "Longest contig": 1972629, "N50 (scaffolds)": 1390797, "N50 (contigs)": 1390797, "Mean scaffold length": 801328.0, "Mean contig length": 801328.0, "Coding density": 0.8865547254058264, "Translation table": 11, "# predicted genes": 5786, "genome_quality": "high_quality"}, {"genome_id": "GCF_014203325.1", "Completeness": 99.46808510638296, "Contamination": 1.536643026004728, "GC": 0.7110368743562213, "GC std": 0.0, "Genome size": 6825909, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 6825909, "Longest contig": 6825909, "N50 (scaffolds)": 6825909, "N50 (contigs)": 6825909, "Mean scaffold length": 6825909.0, "Mean contig length": 6825909.0, "Coding density": 0.8612406054636825, "Translation table": 11, "# predicted genes": 5929, "genome_quality": "high_quality"}, {"genome_id": "GCF_012277335.1", "Completeness": 99.88179669030734, "Contamination": 3.050958760178619, "GC": 0.6810451071301798, "GC std": 0.0, "Genome size": 8045912, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 8045912, "Longest contig": 8045912, "N50 (scaffolds)": 8045912, "N50 (contigs)": 8045912, "Mean scaffold length": 8045912.0, "Mean contig length": 8045912.0, "Coding density": 0.8422994186364454, "Translation table": 11, "# predicted genes": 7504, "genome_quality": "high_quality"}, {"genome_id": "GCF_008630535.1", "Completeness": 99.43853427895982, "Contamination": 2.624113475177305, "GC": 0.7140299510212214, "GC std": 0.0483380434393586, "Genome size": 7549392, "# ambiguous bases": 0, "# scaffolds": 46, "# contigs": 46, "Longest scaffold": 688257, "Longest contig": 688257, "N50 (scaffolds)": 504440, "N50 (contigs)": 504440, "Mean scaffold length": 164117.21739130435, "Mean contig length": 164117.21739130435, "Coding density": 0.8811830144732185, "Translation table": 11, "# predicted genes": 6830, "genome_quality": "high_quality"}, {"genome_id": "GCF_016859185.1", "Completeness": 100.0, "Contamination": 2.6549776727081693, "GC": 0.7111245450688278, "GC std": 0.0, "Genome size": 8305652, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 8305652, "Longest contig": 8305652, "N50 (scaffolds)": 8305652, "N50 (contigs)": 8305652, "Mean scaffold length": 8305652.0, "Mean contig length": 8305652.0, "Coding density": 0.8626696615750334, "Translation table": 11, "# predicted genes": 7407, "genome_quality": "high_quality"}, {"genome_id": "GCF_000062885.1", "Completeness": 100.0, "Contamination": 1.9457578145521408, "GC": 0.7114596048487697, "GC std": 0.0, "Genome size": 8212805, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 8212805, "Longest contig": 8212805, "N50 (scaffolds)": 8212805, "N50 (contigs)": 8212805, "Mean scaffold length": 8212805.0, "Mean contig length": 8212805.0, "Coding density": 0.8636916376317226, "Translation table": 11, "# predicted genes": 7279, "genome_quality": "high_quality"}, {"genome_id": "GCF_014203395.1", "Completeness": 99.29078014184395, "Contamination": 4.078014184397163, "GC": 0.6772580746183299, "GC std": 0.0103248041079479, "Genome size": 8541735, "# ambiguous bases": 0, "# scaffolds": 6, "# contigs": 6, "Longest scaffold": 5889697, "Longest contig": 5889697, "N50 (scaffolds)": 5889697, "N50 (contigs)": 5889697, "Mean scaffold length": 1423622.5, "Mean contig length": 1423622.5, "Coding density": 0.8655130368713148, "Translation table": 11, "# predicted genes": 7445, "genome_quality": "high_quality"}, {"genome_id": "GCF_000194155.1", "Completeness": 98.64066193853428, "Contamination": 2.9977672708169165, "GC": 0.6805661874795961, "GC std": 0.0314472022848502, "Genome size": 8581920, "# ambiguous bases": 54144, "# scaffolds": 22, "# contigs": 117, "Longest scaffold": 3851573, "Longest contig": 471503, "N50 (scaffolds)": 2392754, "N50 (contigs)": 163471, "Mean scaffold length": 390087.2727272727, "Mean contig length": 72886.97435897436, "Coding density": 0.8275158705744169, "Translation table": 11, "# predicted genes": 8230, "genome_quality": "high_quality"}, {"genome_id": "GCF_002564065.1", "Completeness": 100.0, "Contamination": 1.9457578145521408, "GC": 0.7114145473027494, "GC std": 0.0, "Genome size": 8230103, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 8230103, "Longest contig": 8230103, "N50 (scaffolds)": 8230103, "N50 (contigs)": 8230103, "Mean scaffold length": 8230103.0, "Mean contig length": 8230103.0, "Coding density": 0.8635506748821977, "Translation table": 11, "# predicted genes": 7308, "genome_quality": "high_quality"}, {"genome_id": "GCF_003635025.1", "Completeness": 100.0, "Contamination": 3.1855791962174944, "GC": 0.7038623374794615, "GC std": 0.0012732815388598, "Genome size": 8266186, "# ambiguous bases": 0, "# scaffolds": 2, "# contigs": 2, "Longest scaffold": 8125734, "Longest contig": 8125734, "N50 (scaffolds)": 8125734, "N50 (contigs)": 8125734, "Mean scaffold length": 4133093.0, "Mean contig length": 4133093.0, "Coding density": 0.8751350380937473, "Translation table": 11, "# predicted genes": 7658, "genome_quality": "high_quality"}, {"genome_id": "GCF_014490055.1", "Completeness": 99.52718676122932, "Contamination": 3.245994221171526, "GC": 0.6796706110054318, "GC std": 0.0, "Genome size": 8876435, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 8876435, "Longest contig": 8876435, "N50 (scaffolds)": 8876435, "N50 (contigs)": 8876435, "Mean scaffold length": 8876435.0, "Mean contig length": 8876435.0, "Coding density": 0.8446922666588558, "Translation table": 11, "# predicted genes": 8022, "genome_quality": "high_quality"}, {"genome_id": "GCF_002846475.1", "Completeness": 99.52718676122932, "Contamination": 3.245994221171526, "GC": 0.6794950162105907, "GC std": 0.0, "Genome size": 9020646, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 9020646, "Longest contig": 9020646, "N50 (scaffolds)": 9020646, "N50 (contigs)": 9020646, "Mean scaffold length": 9020646.0, "Mean contig length": 9020646.0, "Coding density": 0.8479478077290695, "Translation table": 11, "# predicted genes": 8028, "genome_quality": "high_quality"}, {"genome_id": "GCF_014697215.1", "Completeness": 99.88179669030734, "Contamination": 5.9430935494765285, "GC": 0.6792499724193344, "GC std": 0.0089040739990278, "Genome size": 9608180, "# ambiguous bases": 0, "# scaffolds": 3, "# contigs": 3, "Longest scaffold": 9436548, "Longest contig": 9436548, "N50 (scaffolds)": 9436548, "N50 (contigs)": 9436548, "Mean scaffold length": 3202726.6666666665, "Mean contig length": 3202726.6666666665, "Coding density": 0.8417722190883185, "Translation table": 11, "# predicted genes": 9217, "genome_quality": "medium_quality"}, {"genome_id": "GCF_024734405.1", "Completeness": 99.822695035461, "Contamination": 1.536643026004728, "GC": 0.7266721423436433, "GC std": 0.0, "Genome size": 6344421, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 6344421, "Longest contig": 6344421, "N50 (scaffolds)": 6344421, "N50 (contigs)": 6344421, "Mean scaffold length": 6344421.0, "Mean contig length": 6344421.0, "Coding density": 0.8752644252328148, "Translation table": 11, "# predicted genes": 5540, "genome_quality": "high_quality"}, {"genome_id": "GCF_018070075.1", "Completeness": 99.822695035461, "Contamination": 2.861833464670344, "GC": 0.6975256232823224, "GC std": 0.0319032695050071, "Genome size": 7167017, "# ambiguous bases": 0, "# scaffolds": 34, "# contigs": 34, "Longest scaffold": 858004, "Longest contig": 858004, "N50 (scaffolds)": 363995, "N50 (contigs)": 363995, "Mean scaffold length": 210794.61764705885, "Mean contig length": 210794.61764705885, "Coding density": 0.8839261857478502, "Translation table": 11, "# predicted genes": 6643, "genome_quality": "high_quality"}, {"genome_id": "GCF_022828475.1", "Completeness": 99.05437352245865, "Contamination": 2.423167848699763, "GC": 0.6947996676695216, "GC std": 0.0, "Genome size": 6878695, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 6878695, "Longest contig": 6878695, "N50 (scaffolds)": 6878695, "N50 (contigs)": 6878695, "Mean scaffold length": 6878695.0, "Mean contig length": 6878695.0, "Coding density": 0.8527571000022534, "Translation table": 11, "# predicted genes": 6055, "genome_quality": "high_quality"}, {"genome_id": "GCF_900116135.1", "Completeness": 99.46808510638296, "Contamination": 1.75531914893617, "GC": 0.7077562954321726, "GC std": 0.0373222530498549, "Genome size": 6291299, "# ambiguous bases": 190, "# scaffolds": 22, "# contigs": 24, "Longest scaffold": 1461917, "Longest contig": 1461917, "N50 (scaffolds)": 505605, "N50 (contigs)": 453928, "Mean scaffold length": 285968.13636363635, "Mean contig length": 262129.54166666663, "Coding density": 0.8874002014528319, "Translation table": 11, "# predicted genes": 5631, "genome_quality": "high_quality"}, {"genome_id": "GCF_025643595.1", "Completeness": 99.645390070922, "Contamination": 2.848699763593381, "GC": 0.7170576832060522, "GC std": 0.0, "Genome size": 6562638, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 6562638, "Longest contig": 6562638, "N50 (scaffolds)": 6562638, "N50 (contigs)": 6562638, "Mean scaffold length": 6562638.0, "Mean contig length": 6562638.0, "Coding density": 0.882625858686705, "Translation table": 11, "# predicted genes": 6032, "genome_quality": "high_quality"}, {"genome_id": "GCF_022392385.1", "Completeness": 99.645390070922, "Contamination": 1.9457578145521408, "GC": 0.7118386770561067, "GC std": 0.0015522632597943, "Genome size": 8181083, "# ambiguous bases": 10094, "# scaffolds": 2, "# contigs": 15, "Longest scaffold": 5801759, "Longest contig": 1955749, "N50 (scaffolds)": 5801759, "N50 (contigs)": 1073489, "Mean scaffold length": 4090541.5, "Mean contig length": 544732.6, "Coding density": 0.8639395297664135, "Translation table": 11, "# predicted genes": 7263, "genome_quality": "high_quality"}, {"genome_id": "GCF_018141105.1", "Completeness": 99.645390070922, "Contamination": 2.2826372471762544, "GC": 0.710667418576433, "GC std": 0.0, "Genome size": 8243897, "# ambiguous bases": 0, "# scaffolds": 1, "# contigs": 1, "Longest scaffold": 8243897, "Longest contig": 8243897, "N50 (scaffolds)": 8243897, "N50 (contigs)": 8243897, "Mean scaffold length": 8243897.0, "Mean contig length": 8243897.0, "Coding density": 0.8728290758606033, "Translation table": 11, "# predicted genes": 7435, "genome_quality": "high_quality"}, {"genome_id": "GCF_900108315.1", "Completeness": 99.645390070922, "Contamination": 2.724586288416076, "GC": 0.7089549423325838, "GC std": 0.0724139671090498, "Genome size": 7744048, "# ambiguous bases": 4322, "# scaffolds": 27, "# contigs": 33, "Longest scaffold": 1906593, "Longest contig": 1077102, "N50 (scaffolds)": 729242, "N50 (contigs)": 469672, "Mean scaffold length": 286816.5925925926, "Mean contig length": 234537.15151515152, "Coding density": 0.8804439228682467, "Translation table": 11, "# predicted genes": 7185, "genome_quality": "high_quality"}, {"genome_id": "GCF_900112555.1", "Completeness": 99.52718676122932, "Contamination": 2.724586288416076, "GC": 0.7089995844725187, "GC std": 0.038333430857097, "Genome size": 7777251, "# ambiguous bases": 1592, "# scaffolds": 28, "# contigs": 31, "Longest scaffold": 1963067, "Longest contig": 1200418, "N50 (scaffolds)": 406024, "N50 (contigs)": 406024, "Mean scaffold length": 277758.96428571426, "Mean contig length": 250827.70967741936, "Coding density": 0.8810862604280099, "Translation table": 11, "# predicted genes": 7257, "genome_quality": "high_quality"}, {"genome_id": "GCF_900114905.1", "Completeness": 100.0, "Contamination": 3.1855791962174944, "GC": 0.7043312457011255, "GC std": 0.0367099615787713, "Genome size": 8219321, "# ambiguous bases": 600, "# scaffolds": 43, "# contigs": 47, "Longest scaffold": 1889235, "Longest contig": 1001818, "N50 (scaffolds)": 345124, "N50 (contigs)": 343399, "Mean scaffold length": 191147.0, "Mean contig length": 174866.40425531915, "Coding density": 0.8772927836739799, "Translation table": 11, "# predicted genes": 7662, "genome_quality": "high_quality"}]}}, {"mode": "vega-lite"});
</script>



## References

<font size="2">

{% for i in project().rule_used['checkm']['references'] %}
- *{{ i }}*
{% endfor %}

</font>
